src/Components/Game/index.jsx

```diff
@@ -0,0 +1,128 @@
+import { useEffect, useState } from "react";
+
+import Modal from "react-modal";
+import Choice from "../Choice";
+
+import "./styles.css";
+
+const options = [
+  {
+    id: 0,
+    name: "rock",
+    src: "https://images.vexels.com/media/users/3/145826/isolated/preview/37a6335f8116ec5d233efd8019e2b2c4-round-rock-by-vexels.png",
+    alt: "rock",
+  },
+  {
+    id: 1,
+    name: "paper",
+    src: "https://png.vector.me/files/images/1/8/183047/blank_sticky_note_clip_art.jpg",
+    alt: "paper",
+  },
+  {
+    id: 2,
+    name: "scissor",
+    src: "https://easydrawingguides.com/wp-content/uploads/2020/06/Scissors-Step-10.png",
+    alt: "scissor",
+  },
+];
+
+function getComputerChoice() {
+  let computerChoice = options[Math.floor(Math.random() * 3)];
```

Comment:
```suggestion
  let computerChoice = options[Math.floor(Math.random() * options.length)];
```

We can change that, so instead of hardcoding this to multiply per 3, we multiply to the actual options length. In the future if we add a new option, we won't have to modify this code.

Reply:
do it!